### PR TITLE
Added some changes, please review pr and give suggestions

### PR DESCRIPTION
**Pull request by Aayush-765**

### CODE REVIEW

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@Aayush-765 Lot of changes are required!
1. Images and Videos folder is compulsory. Add it with two sub-folders, "Images" and "Videos" which should contain project related images(except CAD images) and project-related videos(like animations or code stimulation videos).
2. Report is missing inside Poster and Report. A Project Report in pdf format is a must.
3. Poster is not up to the mark. Make sure that you mention "Models and Robotics Section", "Srishti 2020" in your poster. Also, other unnecessary elements should not be present in poster.(Check bottom left corner of your poster).
4. src folder is missing. You should make one folder named "src" and have project related code files(like arduino codes, python codes) inside.
5. README.md is blank. Update it as mentioned in the guideline.

**Comment by Aayush-765** [comment]
Actually  it was not complete earlier, I thought that we could add it one
by one.
But actually there is no working video available, and no one has the bot
right now, we have some images which were clicked earlier.

**Comment by SanjeevKrishnan** [comment]
> Actually it was not complete earlier, I thought that we could add it one by one. But actually there is no working video available, and no one has the bot right now, we have some images which were clicked earlier.
> […](#)


@Shubhanshu07 Please clarify regarding video!

**Comment by SanjeevKrishnan** [comment]
> Actually it was not complete earlier, I thought that we could add it one by one. But actually there is no working video available, and no one has the bot right now, we have some images which were clicked earlier.
> […](#)
@Aayush-765 Try to make some stimulation videos as it will be advantageous to your repo. If it is genuinely not possible to make one then you can skip adding Videos.

**Comment by Shubhanshu07** [comment]
@Aayush-765
1. Write content of readme normal not in italics.
2. Do not use any internet photo.
3. Write team members and mentors as ordered list not  unorders list.
4. Add solid works jped in readme also.
5. For video you can upload bot recognition video using shapes and all.
6. Flow chart is missing.

**Comment by Aayush-765** [comment]
Sir actually I am unable to create a new pull request, since I made one
earlier. Could you help me out with this?

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@Aayush-765 Changes Required!
1. Move Images out of Images and Video folders. As there are no videos, "Images and Videos" is not necessary.
2. Report has to be realigned. Dont break team members into two pages.
3. Also include, "Models and Robotics Section" along with its logo in Report.
4. Poster and Report need not be inside subfolders.
5. Justify text in README.md using HTML tags.

**Comment by SanjeevKrishnan** [comment]
@Aayush-765 Also, you don't have to create a new pull request every time. All your changes get updated in this pull request itself. Just leave a comment here mentioning that "Changes have been made" once your repo is ready!

**Comment by Aayush-765** [comment]
Changes have been made.

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@Aayush-765 Text in REAME.md and Report is yet to be justified.
You can use `<p align="justify">content<\p> `to justify, where content is the text you want to justify.

**Review by SanjeevKrishnan — APPROVED** [review]
@Aayush-765 All requirements are met!
@Shubhanshu07 I suggest for merge!

**Comment by agshubhs** [comment]
@Aayush-765 
Remove  unnecessary space between the point in README

File: README.md [hunk 1]
```diff
@@ -1,2 +1,85 @@
...
+
+| Materials  | Cost(INR) |
+| ------------- | ------------- |
+| 6 DC Servo Motors | 3000 |
+| Arduino UNO R3  |400 |
+|  11.1 Li-Po Battery  | 1800 |
+| Miscellaneous | 300 |
+| Total | 5500 |
+
+## Applications
```
Comment (agshubhs) [inline README.md:49]: Remove p  tag from here

**Comment by Aayush-765** [comment]
Sir please give me 5 mins

**Comment by Aayush-765** [comment]
sir please review now

**Comment by agshubhs** [comment]
@Aayush-765 
PR looks good to me. 
Closing and merging the PR

**Comment by Aayush-765** [comment]
thank you sir

**Pull request merged**